sdk/ai/azopenai/README.md

```diff
@@ -13,7 +13,7 @@ Azure OpenAI Service provides access to OpenAI's features:
 
 ### Prerequisites
 
-* Go, version 1.18 or higher - [Install Go](https://go.dev/doc/install)
+* [Supported](../../../README.md#go-version-support) version of Go - [Install Go](https://go.dev/doc/install)
```

Comment:
Since the README.md files show up on `pkg.go.dev`, I think this will result in a broken link. Can we use an absolute URL here, or maybe have an aka link for our support policy?

Reply:
Sorry I'm not sure what is meant by "switch the branch".